sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/screens/home/HomeScreen.kt

```diff
@@ -19,26 +19,34 @@ import co.nimblehq.sample.compose.ui.screens.AppBar
 import co.nimblehq.sample.compose.ui.theme.ComposeTheme
 import co.nimblehq.sample.compose.ui.userReadableMessage
 import com.google.accompanist.permissions.*
+import kotlinx.coroutines.delay
 
 @Composable
 fun HomeScreen(
     viewModel: HomeViewModel = hiltViewModel(),
     navigator: (destination: AppDestination) -> Unit
 ) {
+    val showLoading: IsLoading by viewModel.showLoading.collectAsState()
+    val uiModels: List<UiModel> by viewModel.uiModels.collectAsState()
+    val firstTimeLaunch: Boolean by viewModel.firstTimeLaunch.collectAsState()
+
     val context = LocalContext.current
     LaunchedEffect(viewModel.error) {
         viewModel.error.collect { error ->
             val message = error.userReadableMessage(context)
             Toast.makeText(context, message, Toast.LENGTH_SHORT).show()
         }
     }
+    LaunchedEffect(firstTimeLaunch) {
+        delay(1000)
+        if (firstTimeLaunch) {
+            Toast.makeText(context, "This is the first time launch", Toast.LENGTH_SHORT).show()
+        }
+    }
     LaunchedEffect(viewModel.navigator) {
```

Comment:
[Optional]
Ditto here with the usage of `collectAsState`
```suggestion
    val destination: AppDestination? by viewModel.navigator.collectAsState(initial = null)
    LaunchedEffect(destination) {
        destination?.let {
            navigator(it)
        }
    }
```

Reply:
The same answer to this one. 🙏🏻

Reply:
Same as [this](https://github.com/nimblehq/android-templates/pull/389#discussion_r1099849312)